Sources/StreamChat/WebSocketClient/EventMiddlewares/MessageReactionsMiddleware.swift

```diff
@@ -1,50 +0,0 @@
-//
-// Copyright © 2021 Stream.io Inc. All rights reserved.
-//
-
-import CoreData
-import Foundation
-
-/// The middleware listens for `EventWithReactionPayload` events and updates `MessageReactionDTO` accordingly.
-struct MessageReactionsMiddleware: EventMiddleware {
-    func handle(event: Event, session: DatabaseSession) -> Event? {
```

Comment:
Are we sure we don't need this anymore? At least with my QA it is working fine, but why did we had this in the first place then? 🤔

Reply:
this is not needed anymore, reactions are written to the database by `EventDataProcessorMiddleware` when a message is part of the event